store/migration/10001__init_schema.sql

```diff
@@ -716,7 +716,9 @@ CREATE TABLE backup_setting (
     day_of_week INTEGER NOT NULL CHECK (
         -1 <= day_of_week
         AND day_of_week < 7
-    )
+    ),
+    -- hook_url is the callback webhook url which will be requested after a successful database backup.
```

Comment:
nit: mention callback and webhook together seems to be redundant. And in our case, since we are not exposing it as a webhook, I would suggest to say callback.

The comment could be more succinct.  "hook_url is the callback url to be requested after a successful backup."